mkat_fpga_tests/test_cbf.py

```diff
@@ -137,7 +139,7 @@ def test_c8n856M4k_channelisation_sfdr_peaks(self):
         Aqf.step('Test Spurious Free Dynamic Range for Wideband Coarse: {}\n'
                  .format(self.DEFAULT_INSTRUMENT))
         self.set_instrument(self.DEFAULT_INSTRUMENT)
-        self._test_sfdr_peaks(required_chan_spacing=290e3, cutoff=53)
+        self._test_sfdr_peaks(required_chan_spacing=209e3, cutoff=53)
```

Comment:
@brickZA The initial value used 290e3, How did you get to that value?
bw/n_chans? if yes how is it 290 and not 209?
Was this a typo?